xgcm/test/test_faceconnections.py

```diff
@@ -7,11 +7,14 @@
 
 @pytest.fixture(scope="module")
 def ds():
-    N = 25
+    N = 25  # TODO: Reduce the size here. I think something like 4 or 5 is sufficient
     return xr.Dataset(
         {
             "data_c": (["face", "y", "x"], np.random.rand(2, N, N)),
-            "u": (["face", "xl", "y"], np.random.rand(2, N, N)),
+            "u": (
+                ["face", "xl", "y"],
+                np.random.rand(2, N, N),
+            ),  # TODO: Will it make testing easier if I make these not random?
```

Comment:
I would use a test function that has some kind of linear increase across the domain, so that it's really obvious when values are in the wrong place, and where they should be.

Making the gradient different along different dimensions (or the magnitude completely different) would help avoid mixing up one dimension with another).

Reply:
Even constant values (as I have overwritten in other tests for development) are helpful to find mistakes, but this does work too. Same as above, I think this could (and should in the interest of minimal test changes) be changed in a dedicated PR though.

Reply:
Okay sure